src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -25,14 +25,17 @@ namespace Microsoft.Health.Fhir.Core.Features.Conformance
     public sealed class SystemConformanceProvider
         : ConformanceProviderBase, IConfiguredConformanceProvider, INotificationHandler<RebuildCapabilityStatement>, IDisposable
     {
+        private readonly int _rebuildDelay = 4 * 60 * 60 * 1000; // 4 hours in milliseconds
```

Comment:
Perhaps?
```suggestion
        private readonly TimeSpan _rebuildDelay = TimeSpan.FromHours(4);
```
Then 
`_rebuildDelay.TotalMilliseconds;`

Reply:
I changed the way this is checking so that when it is disposed it doesn't need to wait the full 4 hours. So this value is still an int, but the delay uses a TimeSpan.